index.html

```diff
@@ -45,6 +45,342 @@ <h2 class="supporting">
                 </ul>
             </span>
         </section>
+        <section class="recent-works">
+            <span class="title-container">
+                <h2>My Recent Works</h2>
+                <hr />
+            </span>
+            <div class="frame-44">
+                <img src="./imgs/Img_Placeholder.png" alt="featured image">
+                <div class="right-block">
+                    <h2>
+                        Multi-Post Stories
+                    </h2>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                        the industry's standard
+                        dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="tech-skills">
+                        <li class="css">
```

Comment:
- [ OPTIONAL ] I noticed here that the space between your text and the list items below is bigger than what is in the Figma Design. I would suggest that even though this is marked as an optional suggestion you implement these changes to make your project look exceptional :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/213869667-5c40a2cc-99a5-4916-a3ad-87037b66c145.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213869701-5f1d069c-fc78-49d7-8689-66811f5f9037.png)

- From the Figma desgn you can see that the elements are closer together :+1: